setup.py

```diff
@@ -161,10 +184,29 @@ def extensions():
     return exts
 
 
+class install(_install):
+    def run(self):
+        build_backend()
+        return super().run()
+
+
+class develop(_develop):
+    def run(self):
+        build_backend()
+        return super().run()
```

Comment:
```suggestion
class install(setuptools.command.install.install):
    def run(self):
        build_backend()
        return super().run()


class develop(setuptools.command.develop.develop):
    def run(self):
        build_backend()
        return super().run()
```

Reply:
That option don't work:
AttributeError: module 'setuptools.command' has no attribute 'install'

Reply:
Then `import setuptools.command.install` before.